@remirror/showcase/src/data/fake-users.ts

```diff
@@ -109,7 +109,7 @@ export const fakeUsers: FakeUserObject = {
       email: 'jacinto.darosa@example.com',
       login: {
         uuid: '85d3419f-484b-48e2-b290-119d75d561de',
-        username: 'purplebird177',
+        username: 'purple_bird177',
```

Comment:
Makes reproducing #215 easier; not necessary for this PR but seems harmless otherwise?